generic/tests/edk2_stability_test.py

```diff
@@ -0,0 +1,46 @@
+import re
+
+from avocado.core import exceptions
+from virttest import error_context
+from virttest import utils_misc
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu edk2 stability test:
+    1) Try to log into a guest
+    2) Check serial log information
+    3) Cycle the above process
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    timeout = params.get_numeric("login_timeout", 40)
+    check_messgae = params.get("check_messgae")
+
+    def boot_check(info):
+        """
+        boot info check
+        """
+        return re.search(info, get_serial_console_output(), re.S)
+
+    def get_serial_console_output():
+        """
+        get the output of serail console
+        """
+        output = vm.serial_console.get_output()
+        return output
+
+    for i in range(int(params.get("reboot_count", 1))):
```

Comment:
params.get_numeric("reboot_count", 1) also work.

Reply:
Thanks for reminding, corrected.